config.json

```diff
@@ -14,6 +14,18 @@
         "initiateLifetime": {
             "hours": 1
         },
+        "loginDir": "/login/",
+        "loginHash": "sha512",
+        "loginIdLength": 24,
+        "loginLifetime": {
+            "minutes": 15
+        },
+        "passwordHash": {
```

Comment:
Not totally sold on `passwordHash`.  This should be renamed, since it is pbkdf2.  It is a type of hashing, but designed to be more secure and "hash" has less meaning than "pbkdf2".